pyproblems/prime_factor_products.py

```diff
@@ -0,0 +1,18 @@
+'''This program is to find out the product of the prime factors of a given number'''
+from pyproblems.utility import is_int
+from pyproblems.prime import is_prime
+
+def product_prime(num1):
```

Comment:
num1 does not appear to be a good naming convention. Rather it can be "number"

Reply:
Understood.